insta/src/App.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import './App.css';
+import PostsPage from './components/PostContainer/PostsPage'
+import Authenticate from './components/Authentication/Authenticate'
+import LoginPage from './components/Login/LoginPage'
+
+const Loggedin = Authenticate(PostsPage);
```

Comment:
Line 7 and 8 isnt necessary since your App is wrapped already wrapped in the Authenticate HOC